blog/_posts/2018-06-13-scala-213-collections.md

```diff
@@ -0,0 +1,211 @@
+---
+layout: blog-detail
+post-type: blog
+by: Julien Richard-Foy
+title: Scala 2.13’s Collections
+---
+
+One more article about the standard collections, really? Indeed, during the last
+18 months a lot of work has been done on the collections and we’ve published
+several blog articles and given several talks to the explain various changes or
+challenges we were facing. This article attempts to summarize **what is going
+to change from an end-user perspective**.
+
+In case you’ve thoroughly followed our previous blog posts and talks, you might
+not learn much from this article. Otherwise, this is the perfect opportunity
+to catch up on the topic in a few minutes!
```

Comment:
you could turn this around a bit, and phrase it more positively. first say it is an introduction for everyone, and then something like "even if you've thoroughly followed our previous blog posts and talks, this article gives you a good overview for summarizing the changes to your colleagues".